doc/xmlrpc_client.py

```diff
@@ -0,0 +1,21 @@
+import xmlrpc.client
+
+HOST = "localhost"
+PORT = "80"
+DB = "postgres"
+USER = "odoo"
+PASS = "odoo"
+
+root = f"http://{HOST}:{PORT}/xmlrpc/"
+
+uid = xmlrpc.client.ServerProxy(root + "common").login(DB, USER, PASS)
```

Comment:
On 15.0 can use

```suggestion
uid = xmlrpclib.ServerProxy(root + "2/common").authenticate(DB, USER, PASS, {})
```

Note: xmlrpclib = client imported from xmlrpc

Reply:
```ImportError: cannot import name 'xmlrpclib' from 'xmlrpc'```